Source/WebKit/UIProcess/API/Swift/WebPage+NavigationDeciding.swift

```diff
@@ -51,7 +51,8 @@ extension WebPage_v0 {
         public var buttonNumber: Int { wrapped.buttonNumber }
 #endif
 
-        var wrapped: WKNavigationAction
+        @_spi(CrossImportOverlay)
```

Comment:
Ditto above, though I think this one would be much harder to move into `WebKit_Internal`, because you'd have to get the latter module to have an associated swiftmodule. I think that would mean a new target.

IMO it's fine to keep this as SPI, though annoying that Swift doesn't provide the same flexibility that clang-based languages do.

Reply:
once WebKit adopts Swift package visibility, this can be switched to package instead of SPI public. I intend to do this later on